tools/admin/src/main/scala/whisk/core/database/UserCommand.scala

```diff
@@ -85,9 +85,7 @@ class UserCommand extends Subcommand("user") with WhiskCommand {
 
     def isUUID(u: String) = Try(UUID.fromString(u)).isSuccess
 
-    def desiredNamespace = Namespace(EntityName(namespace.getOrElse(subject()).trim), authKey.uuid)
-
-    def authKey: AuthKey = auth.map(AuthKey(_)).getOrElse(AuthKey())
+    def desiredNamespace(authKey: AuthKey) = Namespace(EntityName(namespace.getOrElse(subject()).trim), authKey.uuid)
```

Comment:
subject is a `val`, should drop the `()` here and elsewhere.

Reply:
probably makes sense to add the trim on `val subject = ...trim`?

Reply:
`subject` here is of type [ScallopOption[String]][1] and `subject()` invokes the [apply][2] method to extract the underlying value. Hence need to use `subject()` approach

[1]: http://scallop.github.io/scallop/org/rogach/scallop/ScallopOption.html
[2]: http://scallop.github.io/scallop/org/rogach/scallop/ScallopOption.html#apply():A

Reply:
aha.